crates/autopilot/src/arguments.rs

```diff
@@ -100,6 +100,15 @@ pub struct Arguments {
     )]
     pub max_auction_age: Duration,
 
+    /// If a limit order surplus fee is older than this, it will get refreshed.
```

Comment:
Would be nice to mention that this expects a value in seconds.

Reply:
https://github.com/cowprotocol/services/pull/714/commits/e7fb81e968bf4ccaddade9f8aac0407ddd6743cc